client/web/reporter/src/views/Report/Builder.vue

```diff
@@ -827,6 +839,7 @@ export default {
       }).catch(this.toastErrorHandler(this.$t('notification:report.datasources.updateFailed')))
         .finally(() => {
           this.datasources.processing = false
+          this.hideDatasourceConfigurator()
```

Comment:
Its already done in the then part (the closing). You also need to remove the comment above if you removed the hideEvent